### PR TITLE
Fix DeleteBookCommand tests

### LINKED ISSUE #295: Fix DeleteBookCommand test
Fix DeleteBookCommandTest
Fix DeleteBookCommandParserTest

### PR DESCRIPTION
**Pull request by Darkarche3**
Fixed DeleteBookCommandTest
Fixed DeleteBookCommandParserTest
closes #295 

Changed DeleteBookCommand.java
Changed line 71
Changed DeleteCommand to DeleteBookCommand in line 71.
Changed to allow DeleteBookCommandTest and DeleteBookCommandParserTest to pass.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteBookCommand.java [hunk 1]
```diff
@@ -68,7 +68,7 @@ public boolean equals(Object other) {
         }
 
         // instanceof handles nulls
-        if (!(other instanceof DeleteCommand)) {
+        if (!(other instanceof DeleteBookCommand)) {
```
Comment (Darkarche3) [inline src/main/java/seedu/address/logic/commands/DeleteBookCommand.java:71]: Changed from DeleteCommand to DeleteBookCommand.

This allows DeleteBookCommandTest and DeleteBookCommandParserTest to pass.

File: src/test/java/seedu/address/logic/commands/DeleteBookCommandTest.java [hunk 2]
```diff
@@ -73,9 +73,8 @@ public void equals() {
         assertTrue(deleteBookCommand1.equals(deleteBookCommand1));
 
         // same values -> returns true
-        // For some reason, this does not pass
-        // DeleteBookCommand deleteBookCommand1Copy = new DeleteBookCommand(bookStub1);
-        // assertTrue(deleteBookCommand1.equals(deleteBookCommand1Copy));
```
Comment (Darkarche3) [inline src/test/java/seedu/address/logic/commands/DeleteBookCommandTest.java:78]: Deleted comment as issue is resolved.

File: src/test/java/seedu/address/logic/parser/DeleteBookCommandParserTest.java [hunk 3]
```diff
@@ -40,7 +42,7 @@ public void parse_invalidPreamble_failure() {
         assertParseFailure(parser, "i/ string", MESSAGE_INVALID_FORMAT);
     }
 
-    /*@Test
```
Comment (Darkarche3) [inline src/test/java/seedu/address/logic/parser/DeleteBookCommandParserTest.java:43]: Uncommented as issue is resolved.

**Review by jinhanfromNUS — APPROVED** [review]
LGTM!

**Pull request merged**